// vitest/dist/reporters.js
export { B as BasicReporter, d as BenchmarkReportsMap, D as DefaultReporter, a as DotReporter, G as GithubActionsReporter, H as HangingProcessReporter, b as JUnitReporter, J as JsonReporter, R as ReportersMap, c as TapFlatReporter, T as TapReporter, V as VerboseReporter } from './vendor/index.3x3MdmUV.js';
import 'node:fs';
import 'picocolors';
import 'pathe';
import './vendor/tasks.DhVtQBtW.js';
import '@vitest/runner/utils';
import '@vitest/utils';
import './vendor/env.2ltrQNq0.js';
import 'std-env';
import './vendor/utils.BVMrsl6E.js';
import './vendor/base.CTYV4Gnz.js';
import 'node:perf_hooks';
import '@vitest/utils/source-map';
import './vendor/index.BJmtb_7W.js';
import './vendor/global.7bFbnyXl.js';
import './chunks/runtime-console.O41g23Zj.js';
import 'node:stream';
import 'node:console';
import 'node:path';
import './vendor/date.W2xKR2qe.js';
import 'node:os';
import 'node:fs/promises';
import 'execa';
import 'node:url';
import 'path';
import 'fs';
import 'module';
import 'vite';
import 'node:process';
import './vendor/_commonjsHelpers.BFTU3MAI.js';
import 'assert';
import 'events';
import 'node:crypto';
import 'vite-node/utils';
import 'node:module';
